DNN_Platform.build

```diff
@@ -1,6 +1,6 @@
 ﻿<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <WebsitePath>$(RootDirectory)\Website</WebsitePath>
+    <WebsitePath>D:\Websites\dev\dev-972\files</WebsitePath>
```

Comment:
Please don't hardcode this path.

Reply:
Sorry about that. I should be able to use the facepalm emoticon for this.

Reply:
🤦